homeassistant/components/dio_chacon/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""The dio_chacon integration."""
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import DioChaconDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.COVER]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
We should extend our ConfigEntry type to stay typesafe, check airgradient for an example

Reply:
OK, done.